components/TestSubmissionsLink/TestSubmissionsLink.tsx

```diff
@@ -0,0 +1,26 @@
+﻿import React from "react";
+import textStyles from './TestSubmissionsLink.module.scss';
+import Link from "next/link";
+
+interface TestSubmissionsLinkProps {
+    test: string;
+}
+
+export function TestSubmissionsLink(props: TestSubmissionsLinkProps) : JSX.Element {
+    
+    const ref: string = "/test-submissions/"; 
+   
+    //TODO: how will results appear to admin? for now put textarea as placeholder
+
+    return (
+        <li className={textStyles.listItem}>
+            <Link href={ref}>
+                <p className={textStyles.link}>{props.test}</p>
+            </Link>
```

Comment:
what's this link for? where are we linking to?

Reply:
removed